QRCoder/ASCIIQRCode.cs

```diff
@@ -52,7 +52,7 @@ public string[] GetLineByLineGraphic(int repeatPerModule, string darkColorString
                 var lineBuilder = new StringBuilder();
                 for (var x = 0; x < QrCodeData.ModuleMatrix.Count - quietZonesModifier; x++)
                 {
-                    var module = QrCodeData.ModuleMatrix[x + quietZonesOffset][((y + verticalNumberOfRepeats) / verticalNumberOfRepeats - 1)+quietZonesOffset];
+                    var module = QrCodeData.ModuleMatrix[((y + verticalNumberOfRepeats) / verticalNumberOfRepeats - 1)+quietZonesOffset][x + quietZonesOffset];
```

Comment:
This change looks correct to me.  The for loops are correct, with the `x` being the inner loop, but when pulling the data from the matrix, the x/y were swapped.  Compare to ArtQRCode which correctly has `var pixelIsDark = this.QrCodeData.ModuleMatrix[offset + y][offset + x];`

Test hashes need to be updated.

Reply:
Thanks for the review. I fixed the test cases.